man/systemd.special.xml

```diff
@@ -285,15 +286,17 @@
         <term><filename>hibernate.target</filename></term>
         <listitem>
           <para>A special target unit for hibernating the system. This
-          pulls in <filename>sleep.target</filename>.</para>
+          pulls in <filename>sleep.target</filename> and
+          <filename>wakeup.target</filename>.</para>
```

Comment:
I.e. "pulls in" here means "indirectly pulls in".  There's a difference, but I don't think it has any practical effect that makes it worth highlighting in this document.